sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -562,7 +562,8 @@ def test_session_by_conn_str_receive_handler_with_autolockrenew(self, servicebus
 
                     elif len(messages) == 1:
                         print("Starting second sleep")
-                        time.sleep(40)
+                        time.sleep(25) # to ensure that we run out the autolockrenew timeout
+                        time.sleep((session.locked_until - datetime.now()).total_seconds() + 1) # To expire the session.
```

Comment:
so the extra 25s-sleeping is necessary for both the autolocknew and session to expire?

Reply:
Correct.  The two sleeps serve different purposes.